src/main/webapp/app/exercises/programming/hestia/git-diff-report/git-diff-report.component.ts

```diff
@@ -19,9 +19,9 @@ export class GitDiffReportComponent implements OnInit {
 
     @Input() isRepositoryView = false;
 
-    leftCommit: string;
+    leftCommit: string | undefined;
 
-    rightCommit: string;
+    rightCommit: string | undefined;
```

Comment:
This could be written as `rightCommit?: string`.

Maybe you could take care of this, if you edit this in a future PR